tests/unit/package_managers/bundler/test_main.py

```diff
@@ -114,3 +115,83 @@ def test_get_main_package_name_and_version_from_repo_without_origin(
         _get_main_package_name_and_version(package_dir=rooted_tmp_path_repo, dependencies=[])
 
     assert "Failed to extract package name from origin remote" in exc_info.value.friendly_msg()
+
+
+def test__prepare_for_hermetic_build_injects_necessary_variable_into_empty_config(
+    rooted_tmp_path: RootedPath,
+) -> None:
+    expected_config_location = rooted_tmp_path.join_within_root(".bundle/config").path
+    expected_config_contents = (
+        '\nBUNDLE_CACHE_PATH: "${output_dir}/deps/rubygems"\n'
+        'BUNDLE_DEPLOYMENT: "true"\nBUNDLE_NO_PRUNE: "true"\n'
+    )
+
+    assert not expected_config_location.exists(), "Unexpected .bundle/config in rooted_tmp_path"
+
+    result = _prepare_for_hermetic_build(rooted_tmp_path, rooted_tmp_path)
+
+    assert result.template == expected_config_contents
+
+
+def test__prepare_for_hermetic_build_injects_necessary_variable_into_existing_config(
+    rooted_tmp_path: RootedPath,
+) -> None:
+    expected_config_location = rooted_tmp_path.join_within_root(".bundle/config").path
+    expected_config_contents = (
+        '\nBUNDLE_CACHE_PATH: "${output_dir}/deps/rubygems"\n'
+        'BUNDLE_DEPLOYMENT: "true"\nBUNDLE_NO_PRUNE: "true"\n'
+    )
+    existing_preamble = '---\n\nBUNDLER_NONEXISTENT_VARIABLE: "true"\n'
```

Comment:
same as above with `textwrap.dedent` and tripple quotes, it's the pattern I believe we use consistently in most places.

Reply:
Done.